CODEOWNERS

```diff
@@ -1 +1,9 @@
-*   @davidpmccormick @gestchild @jamesgorrie
+# These are default codeowners, later rules take precedence
+*	@wellcomecollection/js-ts-reviewers
+
+# To ensure only properly audited changes are run in CI, we require reviews
+# from @wellcomecollection/buildkite-admin when updating pipeline config
+
+/CODEOWNERS	@wellcomecollection/buildkite-admin
+/.buildkite/    @wellcomecollection/buildkite-admin
+/builds/      	@wellcomecollection/buildkite-admin
```

Comment:
This directory doesn't exist - do we need this?

Reply:
It's a copy from other repos where we have that folder, not needed.